manifests/08-report_overview_crd.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: report-overviews.external-pipeline.redhat.com
+spec:
+  group: external-pipeline.redhat.com
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                low:
+                  type: integer
+                moderate:
+                  type: integer
+                important:
+                  type: integer
+                critical:
+                  type: integer
+  scope: Cluster
+  names:
+    plural: report-overviews
+    singular: report-overview
```

Comment:
It looks like most of the multiword names doesn't use dash, would "reportoverview" make sense ? List crds for example here: http://pastebin.test.redhat.com/901861